index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ol>
+        <li><a href = "./recipes/pizza.html">Pizza</a></li>
+        <li><a href = "./recipes/butterchicken.html">Butter Chicken</a></li>
```

Comment:
`href = "./recipes/butterchicken.html"` => `href="./recipes/butterchicken.html"` 

Remove extra space, same for line no 12